Vagrantfile

```diff
@@ -53,6 +53,51 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
     up_message = <<-MSG
 
+    Linting for Python and React
+    ============================
+
+    Lint Python files with isort, autopep8, and black. Lint React Javascript
+    using eslint. This can be done here using Vim or in your editor of choice.
+    Look at ~/.vimrc to see relevant settings (you will need to configure your
+    editor in a similar way).
+
```

Comment:
Might be helpful to include linting commands here.  So:

```
$ isort path/to/source_file.py
$ autopep8 --in-place path/to/source_file.py
$ black path/to/source_file.py
```

Reply:
Good idea. I never run them like that but it makes sense that somebody would. I'll add them.